src/faction_camp.cpp

```diff
@@ -1266,6 +1266,9 @@ void basecamp::get_available_missions( mission_data &mission_key )
                                   "\n\nRisk: None\n"
                                   "Time: Ongoing" ) );
         mission_key.add( "Assign Jobs", _( "Assign Jobs" ), entry );
+        entry = string_format( _( "Notes:\n"
```

Comment:
Why is there a call to `string_format` without formatting parameters?
```suggestion
        entry = _( "Notes:\nAbandon this camp" );
```